### PR TITLE
Added more options to Testnets and Production newworks

### PR DESCRIPTION
**Pull request by Yasir761**

### CODE REVIEW

**Review by statictype — CHANGES_REQUESTED** [review]

**Comment by cmichi** [comment]
Testnets should be named like this:

image

For Phala it's "Phala Development" and the RPC is `poc5.phala.network`.

File: src/constants/index.ts [hunk 1]
```diff
@@ -10,6 +10,10 @@ export enum RPC {
   SHIBUYA = 'wss://rpc.shibuya.astar.network',
   SHIDEN = 'wss://rpc.shiden.astar.network',
   ALEPHZEROTESTNET = 'wss://ws.test.azero.dev',
+  PEAQ = 'wss://wss.agung.peaq.network',
+  PENDULAM = 'wss://ws.pendulum.network',
+  T3RN = 'wss://ws.t3rn.network',
+  PHALA = 'wss://khala.phala.network/ws',
```
[the remark this answers is not in the document]
Comment (cmichi) [reply src/constants/index.ts:16]: Line 9

File: src/ui/components/sidebar/NetworkAndUser.tsx [hunk 2]
```diff
@@ -46,8 +64,8 @@ export function NetworkAndUser() {
         onChange={e => {
           navigate(`/?rpc=${e}`);
         }}
-        options={options}
-        value={options.find(o => o.value === endpoint)?.value || RPC.CONTRACTS}
+        options={data}
+        value={endpoint}
```
[the remark this answers is not in the document]
Comment (Yasir761) [reply src/ui/components/sidebar/NetworkAndUser.tsx:68]: Ok, I will make change

[the remark this answers is not in the document]
Comment (Yasir761) [reply src/ui/components/sidebar/NetworkAndUser.tsx:68]: I made a change but it showing error : value={options.find((o) => o.value === endpoint)?.value || RPC.CONTRACTS}
Screenshot 2023-03-01 130400

**Pull request closed without merging**